### PR TITLE
Thank you page thanku.html

### PR DESCRIPTION
**Pull request by mansisce**
Hi,
Things which might be an issue : 
 1)  Background image. Let me know if that has to be changed ? Then i will extract it from the psd?
2) Font i am not able to incorporate. I will change it.
3) Let me know if there is any other thing which needs attention.
Regards,
Mansi

### CODE REVIEW

**Comment by kenrick** [comment]
Hey @mansisce you can just repeat the the background `repeat-x` and `repeat-y`

File: thanku.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Tinder for the best gifs on earth">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fun</title>
```
Comment (kenrick) [inline thanku.html]: @mansisce Can your change page title to `Fun - Thank You`

**Comment by mansisce** [comment]
@kenrick  I am doing that. You can check.

File: thanku.html (hunk 1 above)
Comment (mansisce) [inline thanku.html]: done. I need to generate pull request again?

Comment (kenrick) [inline thanku.html]: No just commit and push, It will show up here

**Comment by kenrick** [comment]
@mansisce I believe the master branch was deleted. Please repost this PR to the `gh-pages` branch

**Comment by mansisce** [comment]
Hey master branch i not deleted. Can you be more specific what you want me to do? I have not used git much.

**Comment by kenrick** [comment]
Recreate the PR, but point it to the gh-pages branch

**Comment by mansisce** [comment]
If i am recreating and pointing to gh-pages then i don't have my changes in it. Should i commit to that branch gh-pages and then generate the request?

**Comment by kenrick** [comment]
Sure, you can do that. Pull the `gh-pages` branch locally first. Merge your changes into it. Then push it back up and create a new PR.

**Comment by mansisce** [comment]
check now @kenrick

**Pull request closed without merging**